korman/exporter/material.py

```diff
@@ -719,7 +721,7 @@ def export_cubic_env(self, bo, layer, texture):
         # to a big "finalize" save step to prevent races. The texture cache would
         # prevent that as well, so we could theoretically slice-and-dice the single
         # image here... but... meh. Offloading taim.
-        self.export_prepared_image(texture=texture, owner=layer,
+        self.export_prepared_image(texture=texture, owner=layer, indent=3,
```

Comment:
```suggestion
        self.export_prepared_image(texture=texture, owner=layer
```

This seems to have slipped in